PluginBuilder/Controllers/ApiController.cs

```diff
@@ -55,6 +55,29 @@ public IActionResult GetVersion()
         });
     }
 
+    [AllowAnonymous]
+    [HttpGet("testfunctions")]
+    public async Task<IActionResult> TestFunctions([ModelBinder(typeof(PluginVersionModelBinder))] PluginVersion? btcpayVersion = null)
```

Comment:
What is this for?

Reply:
Oops.. I was testing something and I forgot to remove it. Let me do that now.

Reply:
Removed